prow/scripts/create-dns-entry.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+#In order to run this script you need to use a service account with DNS Administrator role
+
+set -o errexit
+
+if [ -z "$IP_ADDRESS" ]; then
+    echo "\$IP_ADDRESS is empty"
+    exit 1
+fi
+
+if [ -z "$PROJECT" ]; then
+    echo "\$PROJECT is empty"
+    exit 1
+fi
+
+if [ -z "$DNS_ZONE" ]; then
+    echo "\$DNS_ZONE is empty"
+    exit 1
+fi
+
+if [ -z "$DOMAIN" ]; then
+    echo "\$DOMAIN is empty"
+    exit 1
+fi
+
+RANDOM_STRING=$(cat /dev/urandom | env LC_CTYPE=C tr -dc a-z0-9 | head -c 16; echo)
```

Comment:
I saw generating a random string in many places, can you extract it to some tools.sh function?

Reply:
I agree. I also think that we should stick to a "single responsibility principle" here, and NOT generate any names here. This script is for adding DNS records, not for dealing with random numbers.
I would say DNS_NAME should be just given as input to this script.